Server/ScriptCompiler.cs

```diff
@@ -210,9 +210,13 @@ public static bool CompileCSScripts(bool debug, bool cache, out Assembly assembl
 
 			DeleteFiles("Scripts.CS*.dll");
 
-			using (var provider = new CSharpCodeProvider())
-			{
-				var path = GetUnusedPath("Scripts.CS");
+#if !MONO
```

Comment:
Has this actually been tested on mono? We don't want to start adding code that can only be compiled by Roslyn to scripts which in turn will break mono servers.

Reply:
Tested and works.